calculator.py

```diff
@@ -12,15 +12,12 @@ def multiply(self, a, b):
         return a * b
 
     def divide(self, a, b):
```

Comment:
please use better variable name e.g. a can be replaced with variable name like numerator and b can used as denominator.

Reply:
Sure will update. and push back